app/src/main/res/values/strings.xml

```diff
@@ -504,4 +508,12 @@
         <item>144p</item>
     </string-array>
 
+    <!-- App update notification -->
+    <string name="app_update_notification_content_title">NewPipe Update Available</string>
+    <string name="app_update_notification_content_text">Tap to download</string>
+
+    <!-- Flavor names -->
+    <string name="app_flavor_github">github</string>
```

Comment:
I would not place the flavor names here. Making them constants in `App.java` seems to be a good idea.